uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/workbench/widgets/multipage/MultiPageEditor.java

```diff
@@ -38,4 +39,6 @@ void addWidget(final IsWidget widget,
                    final String label);
 
     void clear();
+
+    void addTabBarWidget(final Widget customWidget);
```

Comment:
if possible, better using the type `IsWidget` rather than `Widget`

Reply:
Changed everywhere to use [IsWidget](https://github.com/kiegroup/appformer/blob/dcb0ae16d947b3891efa59af5873a46a0f061931/uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/workbench/widgets/multipage/MultiPageEditor.java#L42).

Reply:
thanks!!! :+1: